File: ServerSide/SAMS.BusinessLogic/Models/StudentAttendances/StudentAttendanceModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.StudentAttendances
{
    public class StudentAttendanceModel
    {
        public int Id { get; set; }
        public int StudentId { get; set; }
        public int SubjectId { get; set; }
        public DateTime Date { get; set; }

        public int NecessaryAttendance { get; set; }
        public int RealAttendance { get; set; }
    }
}

File: ClientSide/src/app/pages/studentAttendances/models/studentAttendanceModel.ts
export class StudentAttendanceModel {

    public id: number;
    public studentId: number;
    public subjectId: number;
    public date: Date;
    public necessaryAttendance: number;
    public realAttendance: number;

    constructor(
        id: number,
        studentId: number,
        subjectId: number,
        date: Date,
        necessaryAttendance: number,
        realAttendance: number,
    ) {
        this.id = id;
        this.studentId = studentId;
        this.subjectId = subjectId;
        this.date = date;
        this.necessaryAttendance = necessaryAttendance;
        this.realAttendance = realAttendance;
    }
}
